<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rfAnalysis</class>
 <widget class="QWidget" name="rfAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radio Frequency Data Analysis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>751</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(28, 0, 101);
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="imageSelectionLabelSidebar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image Selection:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="phantomLabel">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>70</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold
}</string>
     </property>
     <property name="text">
      <string>Phantom:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imagePathInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="phantomPathInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);;
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="roiSidebarLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Region of Interest (ROI) Selection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rfAnalysisLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Radio Frequency Data Analysis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(49, 0, 124);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="exportResultsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Export Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveRoiButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>690</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Save ROI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayNpsButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Display NPS</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="analysisParamsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Analysis Parameter Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="displayMbfButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Display Midband Fit (MBF)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displaySsButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Display Spectral Slope (SS)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displaySiButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Display Spectral Intercept (SI)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editImageDisplayButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Edit Image Display</string>
   </property>
  </widget>
  <widget class="QLabel" name="avMbfLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Av.  MBF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avSsLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>35</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Av.  SS (1e-6)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avSiLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Av.  SI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avMbfVal">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avSsVal">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>35</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avSiVal">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indSiVal">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indMbfVal">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indMbfLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Ind.  MBF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indSiLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Ind.  SI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indSsVal">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>35</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indSsLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>35</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Ind.  SS (1e-6)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseWindowButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose Window to Analyze</string>
   </property>
  </widget>
  <widget class="QFrame" name="legend">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="constructRoiLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>-10</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Radio Frequency Analysis:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="imDisplayFrame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>721</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveDataButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportDataButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Export Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
